source/search-context.puml

```diff
@@ -0,0 +1,32 @@
+ @startuml S-ENDA search context
+   !includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/release/1-0/C4_Context.puml
+   
+   LAYOUT_LEFT_RIGHT
+
+   'Boundary(users, "Users") {
+    '  Person(advanced, "Advanced user")
+    '  Person(user, "General user")
+   '}
+
+    System(msys, "my system")
```

Comment:
what is this? @havardf @mpejcoch

Reply:
I think that was an attempt to get rid of the tripple "=" over that arrow which I added unintentionally. I fix.

Reply:
why do those "=" appear? I got it also now...